share/dictionary/tacacs/dictionary

```diff
@@ -8,7 +8,7 @@
 # Version:	$Id$
 #
 
-PROTOCOL	TACACS		5
+PROTOCOL	TACACS		5	verify=lib
```

Comment:
Does the tacacs dictionary have a validation library? If it does it should be added to the same commit.  Splitting out commits is fine, but if you don't group related changes it makes it very difficult to review.